docs/configuration/http_conn_man/route_config/rate_limits.rst

```diff
@@ -47,25 +42,37 @@ Actions
 
 type
   *(required, string)* The type of rate limit action to perform. The currently supported action
-  types are *service_to_service* , *request_headers* and *remote_address*.
+  types are *source_cluster*, *destination_cluster* , *request_headers*, *remote_address* and
+  *route_key*.
 
-Service to service
-^^^^^^^^^^^^^^^^^^
+Source Cluster
+^^^^^^^^^^^^^^
 
 .. code-block:: json
 
   {
-    "type": "service_to_service"
+    "type": "source_cluster"
   }
 
-The following descriptors are sent:
+The following descriptor entry is appended to the descriptor:
 
-  * ("to_cluster", "<:ref:`route target cluster <config_http_conn_man_route_table_route_cluster>`>")
-  * ("to_cluster", "<:ref:`route target cluster <config_http_conn_man_route_table_route_cluster>`>"),
-    ("from_cluster", "<local service cluster>")
+  * ("from_cluster", "<local service cluster>")
 
 <local service cluster> is derived from the :option:`--service-cluster` option.
 
+Destination Cluster
+^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: json
+
+  {
+    "type": "destination_cluster"
+  }
+
+The following descriptor entry is appended to the descriptor:
+
+  * ("to_cluster", "<:ref:`route target cluster <config_http_conn_man_route_table_route_cluster>`>")
```

Comment:
I would be more verbose here in that it will be the routed cluster, which could be do to "cluster", "weighted_clusters", "cluster_header" etc.